packages/cli/src/controllers/auth.controller.ts

```diff
@@ -138,7 +138,7 @@ export class AuthController {
 		}
 
 		try {
-			user = await this.userService.findOneOrFail({ where: {} });
+			user = await this.userRepository.findOneOrFail({ where: {}, relations: ['globalRole'] });
```

Comment:
in the long run we should aim to move more code out of these controllers, and avoid having repositories as direct dependencies in them.

Reply:
Here I wasn't 100% sure what to do since the service method was a paper-thin wrapper for the repository method.